pkg/cmd/system/info.go

```diff
@@ -154,7 +154,7 @@ func prettyPrintInfoDockerCompat(stdout io.Writer, stderr io.Writer, info *docke
 	// Storage Driver is not really Server concept for nerdctl, but mimics `docker info` output
 	fmt.Fprintf(w, " Storage Driver: %s\n", info.Driver)
 	fmt.Fprintf(w, " Logging Driver: %s\n", info.LoggingDriver)
-	printF(w, " Cgroup Driver:  ", info.CgroupDriver)
+	printF(w, " Cgroup Driver: ", info.CgroupDriver)
```

Comment:
Still has an extra space at the head of the line

Reply:
Yes, but CgroupDriver is under the section `Server`.  The indent is one space.

Reply:
Sorry, I will open a new pr to fix it.